README.md

```diff
@@ -10,12 +10,11 @@ It is inspired by [Pregel](https://research.google/pubs/pub37252/) and [Apache B
 The current interface exposed is one inspired by [NetworkX](https://networkx.org/documentation/latest/).
 
 The main use is for adding **cycles** to your LLM application.
-Crucially, this is NOT a **DAG** framework.
+Crucially, LangGraph is NOT optimized for **DAG** workflows.
```

Comment:
not optimized for ONLY DAG workflows?


as is, sounds like it wont work for dags